src/dispatch/signature/__init__.py

```diff
@@ -123,3 +126,69 @@ def extract_covered_components(result: VerifyResult) -> Set[str]:
         covered_components.add(item.value)
 
     return covered_components
+
+
+def parse_verification_key(
+    verification_key: Optional[Union[Ed25519PublicKey, str, bytes]],
+    endpoint: Optional[str] = None,
+) -> Optional[Ed25519PublicKey]:
+    # This function depends a lot on global context like enviornment variables
+    # and logging configuration. It's not ideal for testing, but it's useful to
+    # unify the behavior of the Dispatch class everywhere the signature module
+    # is used.
+    if isinstance(verification_key, Ed25519PublicKey):
+        return verification_key
+
+    # Keep track of whether the key was obtained from the environment, so that
+    # we can tweak the error messages accordingly.
+    from_env = False
+    if not verification_key:
+        try:
+            verification_key = os.environ["DISPATCH_VERIFICATION_KEY"]
+        except KeyError:
+            return None
```

Comment:
should this be `verification_key = None`? Or could the next line be `from_env = True`?